### ISSUE TITLE
What does step_size means?

### ISSUE THREAD

**Issue by elliayim** [tlc121/FsFont#18]
What is the use of step_size in cfgs/custom.yaml?

**Comment by tlc121** [comment]
it is the step_size of the Stepdecay in optimizer.

**Comment by elliayim** [comment]
Is it responsible for learning rate decay? However, I cannot see any learning rate decay in the tensorboard graphs.

**Comment by tlc121** [comment]
see the line 126 in train.py
